plugins/hydra_submitit_launcher/hydra_plugins/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
```

Comment:
remove this file, it's no longer needed with Hydra 1.0 (and in fact can cause damage because it's incompatible with namespace packages).